powersoftau/Cargo.toml

```diff
@@ -24,15 +26,9 @@ hex-literal = "0.1.4"
 
 [dev-dependencies]
 criterion = "0.3"
-zexe_algebra = { package = "algebra", version = "0.1.0", features = ["parallel", "full"] }
+zexe_algebra = { git = "https://github.com/scipr-lab/zexe", package = "algebra", version = "0.1.0", features = ["parallel", "full"] }
 test-helpers = { path = "../test-helpers" }
 
-# add this to compare performance with old accumulators
```

Comment:
I had initially added this to compare performance with older versions of the accumulator. This is no longer needed.